<?xml version="1.0"?>
<launch>
  <arg name="world_name" default="simple_room" />
  <arg name="region_name" default="path" />

  <include file="$(find social_worlds)/launch/start_world.launch">
    <arg name="world_name" default="$(arg world_name)" />
  </include>

  <node name="spawn_model" pkg="gazebo_ros" type="spawn_model" output="screen"
    args="-file $(find social_worlds)/models/box/model.sdf -sdf -z 1 -model box" />

  <node name="check_$(arg region_name)" pkg="social_worlds" type="check_region_node" output="screen">
    <param name="region_name" value="$(arg region_name)"/>
    <param name="model_name" value="box"/>
  </node>

</launch>
